.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+### Description
+
+<!-- What has changed and why has it changed -->
+
+#### Remaining Tasks
+
+<!-- Tasks left to do, either in this PR or as future work -->
+
+#### Related PRs in other projects (PASQAL developers only)
+
+<!-- Links of others related PR to this one -->
+
+#### Additional merge criteria
+
+<!-- Here, add any extra criteria you consider mandatory for merging on top of the usual criteria -->
+
+#### Breaking changes
+
+<!-- Here, add all the breaking changes that this PR involves if there is any -->
+
+### Checklist
+
+- [ ] The title of the PR follows the right format: [{Label}] {Short Message}. Label examples: IMPROVEMENT, FIX, REFACTORING... Short message is about what your PR changes.
+
+#### Documentation
+
+- [ ] **[TEMPORARY]** — Update the version of pasqal-cloud in `_version.py` following the changes in your PR and by using [semantic versioning](https://semver.org/).
```

Comment:
Question: Why temporary? Do we have a ticket to update this when it stops being?

Reply:
@anneclairelh If you can bring some explanation, I do not have them

Reply:
I also don't think we should write this as temporary, it is irrelevant to the contributor. The only thing that matters is whether it applies right now or not. [Documentation should be timeless.](https://developers.google.com/style/timeless-documentation)

Instead I would suggest indeed to create an issue to get rid of this if this is our goal